src/dom/index.ts

```diff
@@ -126,7 +126,7 @@ function prepareTemplateData(original: Original, idx: number): OriginalData {
     data: {
       id: `${idx + 1}`,
       number: idx + 1,
-      reference: original.referenceId,
```

Comment:
This change breaks a few tests, which will need to be updated.

Reply:
The tests are updated now.